testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/UserTest.java

```diff
@@ -1569,19 +1575,91 @@ public void loginShouldFailAfterPasswordDeleted() {
         loginPage.login(userName, userPass);
         assertTrue("Test user should be successfully logged in.", driver.getTitle().contains("Account Management"));
         accountPage.logOut();
-        
-        Optional<CredentialRepresentation> passwordCredential = 
+
+        Optional<CredentialRepresentation> passwordCredential =
                 realm.users().get(userId).credentials().stream()
-                .filter(c -> CredentialRepresentation.PASSWORD.equals(c.getType()))
-                .findFirst();
+                        .filter(c -> CredentialRepresentation.PASSWORD.equals(c.getType()))
+                        .findFirst();
         assertTrue("Test user should have a password credential set.", passwordCredential.isPresent());
         realm.users().get(userId).removeCredential(passwordCredential.get().getId());
-        
+
         driver.navigate().to(accountUrl);
         assertEquals("Test user should be on the login page.", "Log In", PageUtils.getPageTitle(driver));
         loginPage.login(userName, userPass);
-        assertTrue("Test user should fail to log in after password was deleted.", 
+        assertTrue("Test user should fail to log in after password was deleted.",
                 driver.getCurrentUrl().contains(String.format("/realms/%s/login-actions/authenticate", REALM_NAME)));
     }
-    
+
+    private UserResource findUser(String realm, String username) {
```

Comment:
I think it is possible to remove this method and use already existing method "ApiUtil.findUserByUsername" ?

Reply:
Removed